src/luzmo.js

```diff
@@ -263,8 +263,9 @@ _onConnect() {
       return response.data;
     })
     .catch((error) => {
-      if (!Luzmo._isEmpty(error.response.error)) throw error.response.error;
-      throw error.response;
+      if (!Luzmo._isEmpty(error.response)) throw error.response.data;
+      else if (!Luzmo._isEmpty(error.request)) throw error.request;
+      else throw "An unexpected error occured.";
```

Comment:
I tried to make it throw different types of errors
I don't think the request is useful it doesn't hold any error reason
but for Axios error, it always includes a message, name, code
```suggestion
      if (!Luzmo._isEmpty(error.response)) throw error.response.data;
      delete error.request;
      delete error.config;
      throw error;
```
![image](https://github.com/luzmo-official/luzmo-sdk-nodejs/assets/8212602/4e8e4d00-fb26-4653-ada8-f0513e320779)
![image](https://github.com/luzmo-official/luzmo-sdk-nodejs/assets/8212602/a3307a29-4e25-4018-abc3-237f20098017)